src/backend/base/langflow/api/v1/flows.py

```diff
@@ -299,7 +299,7 @@ async def update_flow(
 
 
 @router.delete("/{flow_id}", status_code=200)
-async def delete_flow(
+def delete_flow(
```

Comment:
We should probably make all endpoints async otherwise FastAPI runs them in a thread. 
Check out number 2 here:
https://github.com/Kludex/fastapi-tips

Reply:
Yes, it’s intentional to run in a thread.
Otherwise delete_flow makes blocking database operations in the event loop which is even worse.